### PR TITLE
fix: remove label about hubic upgrade offer into resiliation pack

### PR DESCRIPTION
**Pull request by Steffy29**
ref: XDSL-5163

### CODE REVIEW

File: client/app/telecom/pack/resiliation/translations/Messages_cs_CZ.xml [hunk 1]
```diff
@@ -36,7 +36,7 @@
...
    <translation id="pack_resiliation_services_cannot_keep" qtlid="313520">En résiliant le pack, vous ne pouvez pas garder ce service.</translation>
    <translation id="pack_resiliation_subservices_load_error" qtlid="313533">Erreur de chargement des informations de résiliation des sous services. Impossible de poursuivre la résiliation.</translation>
-   <translation id="pack_resiliation_hubic_disclamer" qtlid="385676">Vos comptes hubiC seront automatiquement basculés en offre gratuite 25 Go. Si vos données dépassent la capacité de stockage d'une offre gratuite, ces dernières restent accessibles. Vous pourrez changer d'offre à tout moment en vous connectant sur <a target="_blank" href="https://hubic.com/home/" qtlid="313559" qtlid_href="313559">https://hubic.com/home/</a> ou depuis votre application mobile hubiC.</translation>
+   <translation id="pack_resiliation_hubic_disclamer" qtlid="385676">Vos comptes hubiC seront automatiquement basculés en offre gratuite 25 Go. Si vos données dépassent la capacité de stockage d'une offre gratuite, ces dernières restent accessibles.</translation>
```
Comment (JeremyDec) [inline client/app/telecom/pack/resiliation/translations/Messages_cs_CZ.xml]: Remove qtlid if translation has changed please (same of all languages files)

**Comment by antleblanc** [comment]
@Steffy29 You could update only: `client/app/telecom/pack/resiliation/translations/Messages_fr_FR.xml`.

It will be just fine.

**Comment by antleblanc** [comment]
@Steffy29 could you please update your pull request regarding [my previous comment](https://github.com/ovh-ux/ovh-manager-telecom/pull/1087#issuecomment-436570250)?

**Comment by Steffy29** [comment]
> @Steffy29 could you please update your pull request regarding [my previous comment](https://github.com/ovh-ux/ovh-manager-telecom/pull/1087#issuecomment-436570250)?

Yes, I'll do it next week (when I worked with the xDSL team).

**Pull request merged**